indico/modules/events/abstracts/models/abstracts.py

```diff
@@ -367,6 +374,18 @@ def candidate_tracks(self):
         return {t for t in self.reviewed_for_tracks if self.get_track_reviewing_state(t) in states}
 
     @property
+    def edit_track_mode(self):
+        if not inspect(self).persistent:
+            return EditTrackMode.both
+        elif self.state not in {AbstractState.submitted, AbstractState.withdrawn}:
```

Comment:
I don't think it makes sense allowing changing the tracks when the abstract is withdrawn.